profile.js

```diff
@@ -14,6 +17,11 @@ function getUserData(username) {
     }
 }
 
+/**
+ * Build and parse the response from the get request. Sending the desired parsed info to printMessage()
+ * @param res String. Get request response
+ * @param {string} username
+ */
```

Comment:
We should pair on this.
This method is a little weird how its very tightly coupled with the username data.